### ISSUE TITLE
Deploy fails with a CORS configuration error

### ISSUE THREAD

**Issue by jeffogata** [serverless/serverless#6098]
# This is a Bug Report

## Description

* What went wrong?

The following `deploy` command fails on serverless version 1.42.0 with a CORS configuration error.  The same deployment succeeds with version 1.41.1.

`sls deploy --stage $env --region $rgn --package $CODEBUILD_SRC_DIR/artifacts/$env -v`

* What did you expect should have happened?

The deployment should have succeeded, as it did in version 1.41.1.

* What was the config you used?

```
# Welcome to serverless. Read the docs
# https://serverless.com/framework/docs/

# Serverless.yml is the configuration the CLI
# uses to deploy your code to your provider of choice

# The `service` block is the name of the service
service: hello-world-jeff

# The `provider` block defines where your service will be deployed
provider:
  name: aws
  runtime: nodejs8.10

# The `functions` block defines what code to deploy
functions:
  helloWorld:
    handler: handler.helloWorld
    # The `events` block defines how to trigger the handler.helloWorld code
    events:
      - http:
          path: hello-world
          method: get
          cors: true
```

* What stacktrace or error message from your provider did you see?

```
  Serverless Error --------------------------------------- 
  
  You can only use "origin" or "origins", but not both at the same time to configure CORS. Please check the docs for more info. 
  
  Get Support -------------------------------------------- 
     Docs:          docs.serverless.com 
     Bugs:          github.com/serverless/serverless/issues 
     Issues:        forum.serverless.com 
  
  Your Environment Information --------------------------- 
     OS:                     linux 
     Node Version:           8.11.0 
     Serverless Version:     1.42.0 
```

Similar or dependent issues:

## Additional Data

* ***Serverless Framework Version you're using***: 1.42.0
* ***Operating System***: linux
* ***Stack Trace***:
* ***Provider Error messages***:

**Comment by umerthow** [comment]
i got the same issue, but when i downgrade to v. 1.38 , it run smoothly

**Comment by et304383** [comment]
Confirming this issue too.  Seems like a regression test is in order.

**Comment by tgthies** [comment]
Same here!

**Comment by andrit** [comment]
me too

**Comment by jonathanalberghini** [comment]
me too

**Comment by aidinkhosrowshahi** [comment]
it happened to me :


  
  Serverless Error --------------------------------------- 
  
  You can only use "origin" or "origins", but not both at the same time to configure CORS. Please check the docs for more info. 
  
  Get Support -------------------------------------------- 
     Docs:          docs.serverless.com 
     Bugs:          github.com/serverless/serverless/issues 
     Issues:        forum.serverless.com 
  
  Your Environment Information --------------------------- 
     OS:                     linux 
     Node Version:           10.1.0 
     Serverless Version:     1.42.0

**Comment by RaissaBergamini** [comment]
Same problem.

Serverless Error --------------------------------------- 
  
You can only use "origin" or "origins", but not both at the same time to configure CORS. Please check the docs for more info. 
  
  Get Support -------------------------------------------- 
     Docs:          docs.serverless.com 
     Bugs:          github.com/serverless/serverless/issues 
     Issues:        forum.serverless.com 
  
  Your Environment Information --------------------------- 
     OS:                     linux 
     Node Version:           8.11.0 
     Serverless Version:     1.42.0

**Comment by dschep** [comment]
I'm working on a fix, should be able to release it today. for now, use `cors: { origin: '*' }` instead of `cors: true` to work around this bug.

**Comment by dschep** [comment]
Fixed in 1.42.1
